### PR TITLE
[usage] Workspace Pricer uses config

### PR DESCRIPTION
**Pull request by easyCZ**
## Description

Provide config to usage component about how to price each workspace class

## Related Issue(s)

Fixes #

## How to test

1. Run the build directly, as it changes werft config `werft run github -j .werft/build.yaml -f`
2. Check the pod started
3. Check pod logs and validate you're seeing 
```
{"config":{"controllerSchedule":"1h0m0s","creditsPerMinuteByWorkspaceClass":{"default":0.1666666667,"gitpodio-internal-xl":0.3333333333},"server":{"services":{"grpc":{"address":":9001"}}}},"level":"info","message":"Starting usage component.","serviceContext":{"service":"usage","version":"commit-5201aba69e22b07e3d75e0181e19f7711ed987e3"},"severity":"INFO","time":"2022-07-11T08:53:29Z"}
```

## Release Notes

```release-note
NONE
```

## Werft options:

- [x] /werft with-preview

### CODE REVIEW

**Comment by roboquat** [comment]
@easyCZ: Adding the "do-not-merge/release-note-label-needed" label because no release-note block was detected, please follow our [release note process](https://git.k8s.io/community/contributors/guide/release-notes.md) to remove it.

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

File: components/usage/config.json [hunk 1]
```diff
@@ -1,5 +1,9 @@
 {
   "controllerSchedule": "1h",
+  "creditsPerMinuteByWorkspaceClass": {
+    "default": 0.1666666667,
+    "gitpodio-internal-xl": 0.3333333333
```
Comment (andrew-farries) [inline components/usage/config.json:5]: Is this the final name for the workspace class? I think it was agreed that the workspace classes will be called `standard` and `large`.

**Review by andrew-farries — APPROVED** [review]
/hold just in case of the question about workspace class names.

File: components/usage/config.json (hunk 1 above)
Comment (easyCZ) [reply components/usage/config.json:5]: Those are only human names, but the identifier is what's listed here

**Review by Pothulapati — APPROVED** [review]
LGTM :shipit:

**Pull request merged**